runtime/panic.c

```diff
@@ -33,6 +33,7 @@ DECL_PANIC_MESSAGE(PANIC_LIST_STORE, "bad list store");
 DECL_PANIC_MESSAGE(PANIC_MAPPING_STORE, "bad mapping store");
 DECL_PANIC_MESSAGE(PANIC_NO_FILLER, "list has no filler");
 DECL_PANIC_MESSAGE(PANIC_INTERNAL_ERROR, "internal error");
+DECL_PANIC_MESSAGE(PANIC_INVALID_DECIMAL, "invalid decimal");
```

Comment:
@jclark, this is a bug because I have to do this change under #704.

This is the comment that was discussed about this panic code
https://github.com/ballerina-platform/nballerina/pull/704#issuecomment-974738928

Reply:
Shall I change `invalid decimal` into `invalid operation`?